packages/@okta/vuepress-site/conductor.yml

```
@@ -2523,7 +2523,78 @@ redirects:
     to: /docs/guides/set-up-self-service-registration/main/
   - from: /docs/guides/set-up-self-service-registration/next-steps/index.html
     to: /docs/guides/set-up-self-service-registration/main/
-
+  - from: /docs/guides/build-custom-ui-mobile/react-native/before-you-begin/
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/
+  - from: /docs/guides/build-custom-ui-mobile/react-native/create-an-okta-application/
```

```suggestion
  - from: /docs/guides/build-custom-ui-mobile/react-native/create-an-okta-application
```
Also add: from: /docs/guides/build-custom-ui-mobile/react-native/create-an-okta-application/index.html